src/enkf/gridio_gfs.f90

```diff
@@ -3874,6 +3974,7 @@ subroutine writeincrement_pnc(vars3d,vars2d,n3d,n2d,levels,ndim,grdin,no_inflate
                                     ! old logical massbal_adjust, if non-zero
   use_full_hydro = ( ql_ind > 0 .and. qi_ind > 0 .and. &
                      qr_ind > 0 .and. qs_ind > 0 .and. qg_ind > 0 )
+  use_full_hydro = .false.
 
```

Comment:
@emilyhcliu Similar to my previous comment.   Add a comment here to explain why this is hard-coded and comment out lines 3975-3976.

Reply:
Done!